### DISCUSSION TITLE
参考文献作者格式问题

### DISCUSSION THREAD

**Discussion by SongtaoMao** [sjtug/SJTUThesis#807] [提问] [answered]
参考文献作者格式中设置
\usepackage[backend=biber,style=gb7714-2015,gbnamefmt=lowercase, sorting=gb7714-2015]{biblatex}
时，作者的格式为姓在前，名在后缩写，如Amiot C（Amiot 是姓，C是名的缩写）
我希望改成作者的格式为名在前缩写，姓在后，IEEE格式（见https://www.overleaf.com/learn/latex/Biblatex_bibliography_styles）
直接改成\usepackage[backend=biber,style=ieee,gbnamefmt=lowercase, sorting=gb7714-2015]{biblatex}为什么不行

**Comment by LogCreative** [comment]
考虑只使用 IEEE 格式呢
```latex
\usepackage[backend=biber,style=ieee]{biblatex}
```

> **注意** 应当遵循国标。

**Reply by SongtaoMao** [reply]
这样会报错 把style改成任何模板里均不行

**Comment by hushidong** [comment, marked as answer]
gbnamefmt=givenahead
